dis_snek/models/enums.py

```diff
@@ -534,7 +534,7 @@ class ActivityType(IntEnum):
 
 
 class Status(str, Enum):
-    """Represents the statuses a user may have."""
+    """Represents the status' a user may have."""
```

Comment:
Errant apostrophe

Reply:
Disagree, it's valid in punctuation and in it's contextually derived use it's proper for plural representation of how many status is possessed.

Reply:
In English statuses is acceptable as per the Oxford Dictionary

Reply:
Well I think Oxford is wrong >:(

Reply:
![](https://media.giphy.com/media/L3X9GvVhP1nY23Ah6u/giphy.gif)

Reply:
This PR wont be merged until this is reverted

Reply:
ok ok fine :(